### ISSUE TITLE
Reduce conversions calls?

### ISSUE THREAD

**Issue by chasemc** [mwang87/MassSpecWebConversion#2]
Are all these separate calls necessary? Or could it be reduced to 1? 
https://github.com/mwang87/MassSpecWebConversion/blob/705de63a549381c7195ee318520eac999f3465f6/util.py#L88-L111

**Comment by mwang87** [comment]
Yeah probably could be collapsed to the Thermo call. I was thinking i might need to do lock mass but apparently only Waters supports it.
